src/m365/aad/commands/app/app-add.ts

```diff
@@ -195,6 +213,91 @@ class AadAppAddCommand extends GraphCommand {
     return request.post<AppInfo>(createApplicationRequestOptions);
   }
 
+  private grantAdminConsent(appInfo: AppInfo, logger: Logger): Promise<AppInfo> {
+    return this.createServicePrincipal(appInfo.appId)
+      .then((sp: ServicePrincipalInfo) => {
+        if (this.debug) {
+          logger.logToStderr("Service principal created, returned object id: " + sp.id);
+        }
+
+        const tasks: Promise<void>[] = [];
+
+        this.appPermissions.forEach(permission => {
+          if (permission.scope.length > 0) {
+            tasks.push(this.grantOAuth2Permission(sp.id, permission.resourceId, permission.scope.join(' ')));
+            
+            if (this.debug) {
+              logger.logToStderr(`Admin consent for following resource ${permission.resourceId}, with delegated permissions: ${permission.scope.join(',')}`);
```

Comment:
We're missing a verb here: 'Granting admin consent'?

Reply:
More "Admin consent granted for", as it's already done when displayed.